packages/calcite-components/src/components/tooltip/tooltip.e2e.ts

```diff
@@ -1090,18 +1146,26 @@ describe("calcite-tooltip", () => {
     expect(await tooltip1.getProperty("open")).toBe(false);
     expect(await tooltip2.getProperty("open")).toBe(false);
 
-    await page.$eval("#ref1", (el: HTMLElement) => {
-      el.dispatchEvent(new PointerEvent("pointermove"));
-    });
+    await page.$eval(
```

Comment:
Emitting pointer events could be generalized since there are a few instances of it.